### PR TITLE
Implemented info command in lage v2

### PR DESCRIPTION
**Pull request by briansilah**
Implemented the info command in lage v2

### CODE REVIEW

File: packages/cli/src/commands/info/index.ts.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Command } from "commander";
+import { infoAction } from "./action.js";
+import { addFilterOptions } from "../addFilterOptions.js";
+
+const infoCommand = new Command("info");
```
Comment (kenotron) [inline packages/cli/src/commands/info/index.ts.ts]: Please add a description here - something like this:

"(UNSTABLE) The info command displays information about a target graph in a workspace. The generated output can be read by other task runners, such as BuildXL."

**Review by kenotron — COMMENTED** [review]
Please add the description for the info command

File: packages/cli/src/commands/info/action.ts [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    dependencies: target.dependencies,
+    workingDirectory,
+    package: target.packageName,
+    task: target.task,
+  };
+
+  return packageTask;
+}
+
+function generateCommand(target, config) {
```
Comment (unpervertedkid) [inline packages/cli/src/commands/info/action.ts:167]: @kenotron Maybe you can give me some pointers on this.
Wasn't really sure on how to implement this.

Comment (kenotron) [reply packages/cli/src/commands/info/action.ts:172]: this looks alright... it's not perfect, but we'll start here.

**Pull request merged**